src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -154,6 +154,38 @@
  * &lt;module name=&quot;VisibilityModifier&quot;/&gt;
  * </pre>
  * <p>
+ * Example with default values:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ * private int myPrivateField1;
+ *
+ * int field1;               // violation, must have a visibility modifier
+ *
+ * protected String field2; // violation, protected visibility is not allowed
+ *
+ * public int field3 = 42; // violation, not static final, immutable, matching the pattern
```

Comment:
Please do vertical alignment of violation trailing comments

Reply:
done